Applications/Games/Steam/Online/script.js

```diff
@@ -10,6 +12,13 @@ var installerImplementation = {
             .url("https://steamcdn-a.akamaihd.net/client/installer/SteamSetup.exe")
             .checksum("4b1b85ec2499a4ce07c89609b256923a4fc479e5")
             .category("Games")
+            .wineVersion("4.8") // HOTFIX: https://github.com/PhoenicisOrg/scripts/issues/881
+            .wineDistribution("staging")
```

Comment:
Is staging really required ?

Reply:
@ImperatorS79 if you tell me how to set winxp environment on `steamwebhelper.exe` i can test upstream.

Reply:
You already set the wine version for the whole prefix. You can also set it only for specific exe, example:
```js
wine.setOsForApplication().set("EtherOne-Win32-Shipping.exe", "win7").do();
```
But this has nothing to do with upstream vs staging.

Reply:
perfect

Reply:
So, why staging and not upstream ?